### ISSUE TITLE
[Chore] Deploying production environment

### ISSUE THREAD

**Issue by SProjects** [andela/vof-deployment-scripts#12]
**What is the issue?**
The production version of VOF has not been deployed to GCP. Force converting HTTP -> HTTPS traffic is the blocker keeping this from happening. 

**Workaround**
N/A

**Debug log output**
N/A

**Expected Behavior**
Once the VOF `develop` branch is merged into `master`, the deployment pipeline will automatically deploy to the production environment OR adding the CircleCI configuration file directly to the `master` branch and making a PR to the main vof-tracker project.

Use this vof-tracker ![fork](https://github.com/SProjects/vof-tracker) since it already contains the latest CircleCI config file or your make your own from the main ![vof-tracker project](https://github.com/andela/vof-tracker)

**Steps to reproduce**
N/A
